util/storage.go

```diff
@@ -96,7 +99,7 @@ func (s *Storage) save() error {
 func (s *Storage) Get(key string) (string, error) {
 	value, ok := s.data[key]
 	if !ok {
-		return "", ErrKeyNotFound
+		return "", &ErrKeyNotFound{key}
```

Comment:
There's no place this would accidentally print out a private key or anything right?

Reply:
It's just the name of the account, eg. "owner", "operator", etc.